### PR TITLE
Add support for Visual Studio 2022.

### PR DESCRIPTION
**Pull request by venkify**
Added initial support for Visual Studio 2022.
Tested with recent trunk with both 2019 and 2022. 

2 things that may need cleanup :
1. Getting Visual Studio path seems a bit tricky. Used my local path for now.
2. VSIX Manifest needs comment/uncomment depending on version building for.

### CODE REVIEW

File: UnityMixedCallstack.csproj [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">11.0</VisualStudioVersion>
-    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
     <!-- <VSSDK140Install Condition="'$(VSSDK140Install)' == ''">$(MSBuildProgramFiles32)\Microsoft Visual Studio 14.0\VSSDK\</VSSDK140Install> -->
-    <VSSDK170Install Condition="'$(VSSDK170Install)' == ''">$(MSBuildProgramFiles32)\Microsoft Visual Studio\2019\Professional\VSSDK\</VSSDK170Install>
+    <VSSdkPath Condition="'$(VisualStudioVersion.Contains(`17`))'">$(ProgramFiles)\Microsoft Visual Studio\2022\VSSDK\</VSSdkPath>
```
Comment (UnityAlex) [inline UnityMixedCallstack.csproj]: For me this is in: `C:\Program Files\Microsoft Visual Studio\2022\Professional\VSSDK` -- what version of VS are using?

**Comment by UnityAlex** [comment]
Thanks for doing this I haven't found time to do this myself! I'll be happy to upload a visix installer for 2022 once we get this working.

File: UnityMixedCallstack.csproj (hunk 1 above)
Comment (venkify) [reply UnityMixedCallstack.csproj]: I installed 2022 Professional to a custom location. (just removed "Professional" from the default path). We can set it back to default path. Couldn't find a registry or other way to get this info (now that vswhere is needed to locate VS installations.)

Comment (UnityAlex) [reply UnityMixedCallstack.csproj]: Yeah I also briefly ran down the rabbit hole of attempting to automagically determine install location without much success. If you could change this to the default path that would be great! Once this lands I'll upload a visix installer for VS2022 as a release here.

Comment (venkify) [reply UnityMixedCallstack.csproj]: Changed to default paths in https://github.com/Unity-Technologies/UnityMixedCallstack/pull/14/commits/458bcd4ebae0f495d0fc0328773cacca20ab1577

**Review by UnityAlex — APPROVED** [review]

**Pull request merged**